sdk/src/cose_validator.rs

```diff
@@ -25,40 +26,43 @@ use x509_parser::{
     prelude::*,
 };
 
+#[cfg(feature = "openssl_sign")]
```

Comment:
We should declare this under "openssl" instead of "openssl_sign" since it is for verifying, not just signing. I've been making that change in some other places.